docs/content/users/extend/additional-services.md

```diff
@@ -67,6 +67,10 @@ Officially-supported add-ons:
 * [Selenium Standalone Chrome](https://github.com/ddev/ddev-selenium-standalone-chrome): `ddev get ddev/ddev-selenium-standalone-chrome`.
 * [Varnish](https://github.com/ddev/ddev-varnish): `ddev get ddev/ddev-varnish`.
 
+## Viewing and Removing Installed Add-Ons
+
+Add-ons installed in DDEV v1.22+ are versioned and can be viewed with `ddev get --installed`. Add-ons can be removed with `ddev get --remove <addonname>`. If you have add-ons that were installed before v1.22, just re-add them with `ddev get <addonname>` and they will be versioned and available in `ddev get --installed`.
```

Comment:
[Run commands, avoid “just”](https://ddev.readthedocs.io/en/stable/developers/writing-style-guide/#voice-and-tone):

```suggestion
Add-ons installed in DDEV v1.22+ are versioned and can be viewed by running `ddev get --installed`. You can remove an add-on by running `ddev get --remove <addonname>`. If you have add-ons that were installed before v1.22, re-add them with `ddev get <addonname>` and they will be versioned and available in `ddev get --installed`.
```